src/structures/ClientPresence.js

```diff
@@ -9,10 +9,6 @@ const { ActivityTypes, Opcodes } = require('../util/Constants');
  * @extends {Presence}
  */
 class ClientPresence extends Presence {
-  /**
-   * @param {Client} client The instantiating client
-   * @param {APIPresence} [data={}] The data for the client presence
```

Comment:
If `APIPresence` doesn't need to be documented in the lone private method `_parse()` below, the `@external` could be removed too?

Reply:
I think that docstring provides some value, so I'm opting to keep it.
I removed all the other things as per your suggestions however, thank you for these!